core/src/main/scala/org/apache/spark/api/java/JavaDoubleRDD.scala

```diff
@@ -19,6 +19,8 @@ package org.apache.spark.api.java
 
 import java.lang.{Double => JDouble}
 
+import org.apache.commons.collections.ComparatorUtils
```

Comment:
I don't think you need this and the project doesn't depend directly on commons-collections anyway IIRC.